manifests/00-crd-operator-config.yaml

```diff
@@ -13,6 +13,32 @@ spec:
   subresources:
     status: {}
   versions:
-  - name: v1
-    served: true
-    storage: true
+    - name: v1
+      served: true
+      storage: true
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            managementState:
+              pattern: ^(Managed|Unmanaged|Removed|Forced)$
+              type: string
+              description: managementState indicates whether and how the operator
+                should manage the component
+            authentication:
+              properties:
+                logoutRedirect:
```

Comment:
Late comment, but just realized, `authentication.logoutRedirect` is on `console.config.openshift.io`, not `console.operator.openshift.io` (the top level user-facing console config, not the config for the operator).  This exists here:  https://github.com/openshift/cluster-config-operator/blob/master/manifests/0000_05_config-operator_01_console.crd.yaml

- [ ] We should remove this, PR it there against that file.
- [ ] Also, lets ensure it handles `""` / unset as these are unset by default and optional.